package.json

```diff
@@ -23,7 +23,7 @@
     "eject": "react-native-scripts eject",
     "android": "react-native-scripts android",
     "ios": "react-native-scripts ios",
-    "lint": "./node_modules/.bin/eslint .",
+    "lint": "./node_modules/.bin/eslint . --fix",
```

Comment:
I haven't tested this yet, but I'm not sure how I feel about this changing code. In some cases we have a valid reason to ignore lint rules, and I know I often forget to add those in until lint flags them.

If it's just adding commas, removing whitespace or changing tabs to 2 spaces then cool. After that I'm wary.

If you and the rest of the app team think your change is preferable and won't cause any issues then all good. Otherwise, I'd like it to be a separate rule. At least we should consider updating travis scripts to not make "fix" changes.

What do you think?